examples/seq2seq/run_eval.py

```diff
@@ -109,23 +137,42 @@ def run_generate():
         fp16=args.fp16,
         task=args.task,
         prefix=args.prefix,
-        **parsed,
+        **parsed_args,
     )
+
     if args.reference_path is None:
         return
+
     # Compute scores
     score_fn = calculate_bleu if "translation" in args.task else calculate_rouge
     output_lns = [x.rstrip() for x in open(args.save_path).readlines()]
     reference_lns = [x.rstrip() for x in open(args.reference_path).readlines()][: len(output_lns)]
     scores: dict = score_fn(output_lns, reference_lns)
     scores.update(runtime_metrics)
-    print(scores)
+
+    dump_args = {}
+    if args.dump_args:
+        dump_args = parsed_args
+    if args.info:
+        dump_args["info"] = args.info
+
+    if verbose:
+        outputs = [scores]
+        if dump_args:
+            outputs.append(dump_args)
+        print(*outputs)
+
     if args.score_path is not None:
-        json.dump(scores, open(args.score_path, "w"))
-    return scores
+        path = args.score_path
+        json.dump(scores, open(path, "w"))
+        if dump_args:
+            path = path.replace(".json", ".args.json")
+            json.dump(dump_args, open(path, "w"))
```

Comment:
why to files? Does this make your life easier?
Can we just write one big dictionary to one json file?

Reply:
Oh, of course, I was just guessing. I had no idea how this dump file is being used, so I didn't want to break anything should someone rely on its current behavior.

Do you propose to merge them all into one dict?

Reply:
yes!